### PR TITLE
Add GHA CI workflow

### PR DESCRIPTION
**Pull request by arueckauer**
|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

Replace TravisCI build with GitHub Action CI workflow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # mezzio-tooling
 
-[![Build Status](https://travis-ci.com/mezzio/mezzio-tooling.svg?branch=master)](https://travis-ci.com/mezzio/mezzio-tooling)
-[![Coverage Status](https://coveralls.io/repos/github/mezzio/mezzio-tooling/badge.svg?branch=master)](https://coveralls.io/github/mezzio/mezzio-tooling?branch=master)
+[![Build Status](https://github.com/mezzio/mezzio-tooling/actions/workflows/continous-integration.yml/badge.svg)](https://github.com/mezzio/mezzio-tooling/actions/workflows/continous-integration.yml)
```
Comment (froschdesign) [inline README.md]:
```suggestion
[![Build Status](https://github.com/mezzio/mezzio-tooling/actions/workflows/continuous-integration.yml/badge.svg)](https://github.com/mezzio/mezzio-tooling/actions/workflows/continuous-integration.yml)
```
You can check the preview via "View file" if the image file works: https://github.com/mezzio/mezzio-tooling/blob/007d8f8a7f976a87511759205c306a478f2ba258/README.md

Comment (arueckauer) [reply README.md]: Good catch!

File: composer.json [hunk 2]
```diff
@@ -27,6 +27,7 @@
     "require": {
         "php": "^7.1",
```
Comment (froschdesign) [inline composer.json:28]: You can use PHP version 7.3 here because of #7

Comment (arueckauer) [reply composer.json:28]: Done.

**Pull request merged**